src/components/SimpleCommonComponents/DynamicTooltip/index.js

```diff
@@ -31,27 +32,25 @@ const _DataProvider = (
 
       /* eslint-disable react/prop-types */
       const message = () => (
-        <>
-          <b>{accession}</b>
-          <br />
-          {name} <br />
-          <small>{databases[db].name}</small>
+        <section className={f('entry-popup')}>
+          <h6>
+            {databases[db].name} - {accession}
+          </h6>
```

Comment:
Is this tooltip used for anything besides domain architectures? If no, maybe it is not necessary to display `databases[db].name`.

Reply:
Mhh... the user can still choose between InterPro or Pfam, so would leave it there